shenyu-admin/src/test/java/org/apache/shenyu/admin/service/register/FallbackShenyuClientRegisterServiceTest.java

```diff
@@ -72,7 +77,12 @@ static class MockFallbackShenyuClientRegisterServiceException extends FallbackSh
         String doRegisterURI(final String selectorName, final List<URIRegisterDTO> uriList) {
             throw new ShenyuException("Exception");
         }
-
+        
+        @Override
+        String doHeartbeat(final String selectorName, final List<URIRegisterDTO> uriList) {
+            throw new ShenyuException("Exception");
```

Comment:
why exception?

Reply:
this is a Exception class, for Exception test, copied from org.apache.shenyu.admin.service.register.FallbackShenyuClientRegisterServiceTest.MockFallbackShenyuClientRegisterServiceException#doRegisterURI